features/steps/cli_steps.py

```diff
@@ -391,9 +391,12 @@ def exec_notebook(context, command):
 @then('I wait for the jupyter webserver to run for up to "{timeout:d}" seconds')
 def wait_for_notebook_to_run(context, timeout):
     timeout_start = time()
+    # FIXME: Will continue iterating after the process has returned
```

Comment:
Should this comment stay? 👀

Reply:
It's stating that there's a flaw in the current logic, but I didn't have time to fix it 😅